File: c11_svg/B_23_Petar_Georgiev.rb
def gime_a_circle_green x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="' +r.to_s+'"
	stroke="green" stroke-width="4" fill="green"/>'
end
def gime_a_circle_red x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="' +r.to_s+'"
	stroke="red" stroke-width="4" fill="red" />'
end
def gime_a_circle_yellow x,y,r
	'<circle cx="'+x.to_s+'" cy="'+y.to_s+'" r="' +r.to_s+'"
	stroke="yellow" stroke-width="4" fill="yellow" />'
end

def rectangle x,y,w,h
	'<rect width="'+w.to_s+'" height="'+h.to_s+'"
	style="fill:rgb(0,0,0);stroke-width:3;stroke:rgb(0,0,0)"/>'
end

def name x,y
  '<text x="'+x.to_s+'" y="'+y.to_s+'" fill="red">Petar Georgiev</text>'
end

File.open("result.svg","w") do |f|
	f.write('<svg xmlns="http://www.w3.org/2000/svg">')
	f.write(rectangle 0,0,60,140)
	f.write(gime_a_circle_red 30,20,20)
	f.write(gime_a_circle_yellow 30,70,20)
	f.write(gime_a_circle_green 30,120,20)
	f.write(name 0,160)
	f.write('</svg>')
end

File: c21/B_23_Petar_Georgiev.js
var box_width = 20,box_height = 20,x=20,y=20;
var space_between = 20;
var i,j;
var array = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,];
var count = 0;
for(i=0;i<=(box_width+space_between)*4;i+=box_width+space_between){
    for (j=0;j<=(box_height+space_between)*4;j+=box_height+space_between){
        fill(255,255,255);
        rect(x+i,y+j,box_width,box_height);
        fill(0,0,0);
        text(array[count],x+i+6,y+j+6,16,16);
        count++;
    }
}
